frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -985,7 +999,13 @@ export const experimentLogic = kea<experimentLogicType>([
             actions.refreshExperimentResults(true)
         },
         resetRunningExperiment: async () => {
-            actions.updateExperiment({ start_date: null, end_date: null, archived: false })
+            actions.updateExperiment({
+                start_date: null,
+                end_date: null,
+                archived: false,
+                conclusion: null,
+                conclusion_comment: null,
+            })
```

Comment:
We'll see what feedback we get from users, because it'll be nice to have a way to preserve a conclusion if the experiment had any. But I'm okay with removing them.